src/helpers/desaturate.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import parseColorString from '../internalHelpers/_parseColorString'
+import rgbToHsl from '../internalHelpers/_rgbToHsl'
+import hslToHex from '../internalHelpers/_hslToHex'
+import hslToRgb from '../internalHelpers/_hslToRgb'
+import guard from '../internalHelpers/_guard'
+
+/**
+ * Returns a string value for the desaturated color.
```

Comment:
This is a nitpick, but I feel like this description should explain what desaturation is rather than say that it returns a string. That's obvious from the usage examples and the type signature anyway. (same goes for saturation)